### ISSUE TITLE
Package-View broken

### ISSUE THREAD

**Issue by MarkusS04** [go-gitea/gitea#29851]
### Description

When opening the Packages, the Rrpo doesn't matter I get the following error:
Also in the admin package.

```
Render failed, failed to render template: repo/packages, error: template error: builtin(bindata):package/shared/list:23:16 : executing "package/shared/list" at <.FullWebLink>: can't evaluate field FullWebLink in type *packages.PackageDescriptor
----------------------------------------------------------------------
					<a href="{{.FullWebLink}}">{{.Package.Name}}</a>
					           ^
----------------------------------------------------------------------
```

### Gitea Version

1.21.8 built with GNU Make 4.4.1, go1.21.8 : bindata, timetzdata, sqlite, sqlite_unlock_notify

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/89395177/ade02c57-4bbd-49bb-8b54-23094397e8cc)

### How are you running Gitea?

Docker Container

### Database

PostgreSQL

**Comment by wxiaoguang** [comment]
Sorry for the inconvenience.

* Known Issues
    * Browsing "packages" pages might cause 500 error, it could be fixed by updating to pre-1.21.9:
        * Docker image: use `gitea/gitea:1.21-nightly` and `gitea/gitea:1.21-nightly-rootless`
        * Binary: download from https://dl.gitea.com/gitea/1.21/

**Comment by silverwind** [comment]
Duplicate of pinned issue https://github.com/go-gitea/gitea/issues/29773.
